multi-node/aws/cmd/kube-aws/command_validate.go

```diff
@@ -34,6 +34,13 @@ func runCmdValidate(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("Unable to load cluster config: %v", err)
 	}
 
+	fmt.Printf("Validating Key Pair...\n")
```

Comment:
Move this logic to https://github.com/coreos/coreos-kubernetes/blob/master/multi-node/aws/pkg/cluster/cluster.go#L56